common/cb_types.txt

```diff
@@ -1549,7 +1549,7 @@ liberate_and_puppet_country = {
 	months = 12
 
 	badboy_factor = 18
-	prestige_factor = 1
+	prestige_factor = 0.9
```

Comment:
Just noticed that this is not `peace_cost_factor`. Is that intentional?

Reply:
Oops